### ISSUE TITLE
A/T Disco button shouldn't be able to reconnect A/T

### ISSUE THREAD

**Issue by MCCVeen** [mSparks43/747-400#541]
Whenever I turn off the Autothrottle Disconnect button on the yoke, I press the autothrottle disconnect button twice to get rid of the >AUTOTHROT DISC message on the Eicas. However when I press the disco button when the autothrottle is already disconnect, it turns it back on which isn't supposed to happen.

**Comment by Matt726-S** [comment]
> FCOM says
> ![image](https://user-images.githubusercontent.com/6773686/117874134-eae01e80-b2a0-11eb-8e20-86711f86bd26.png)

yes, it remains armed. there’s a difference between on and armed. armed means its just ready to be activated. when the autothrottle is disconnected with the throttle button, it remains off, but the pilot should be able to move it by themself. according to fcom, the autothrottle activates in these situations:
![11ADE748-85CB-4C47-AA13-169FE51E52D8](https://user-images.githubusercontent.com/62818033/117875247-f109d700-b26f-11eb-8eb3-bd44269e3d35.jpeg)

**Comment by mSparks43** [comment]
second press only rearms it (clears the fail flag which locks out mode changes)
something else is activating it

**Comment by Matt726-S** [comment]
> second press only rearms it (clears the fail flag which locks out mode changes)
> something else is activating it

second push should only clear the message. please fix this before release because people who know that they have to push the button twice will be confused when the autothrottle re engages during landing, then since the auto throttle is on they will mess up their landing

**Comment by mSparks43** [comment]
> > second press only rearms it (clears the fail flag which locks out mode changes)
> > something else is activating it
> 
> second push should only clear the message. please fix this before release because people who know that they have to push the button twice will be confused when the autothrottle re engages during landing, then since the auto throttle is on they will mess up their landing

you posted the fcom image that says "second push:autothrottle remains armed"

something else is engaging it - need to know what to fix it.
ill put this as a second release issue tho

**Comment by MCCVeen** [comment]
> > > second press only rearms it (clears the fail flag which locks out mode changes)
> > > something else is activating it
> > 
> > 
> > second push should only clear the message. please fix this before release because people who know that they have to push the button twice will be confused when the autothrottle re engages during landing, then since the auto throttle is on they will mess up their landing
> 
> you quoted the fcom image that says "second push:autothrottle remains armed"
> 
> something else is engaging it - need to know what to fix it.
> ill put this as a second release issue tho

I think it'll clear up some confusion if you have a look at #591
